README.md

```diff
@@ -359,297 +359,372 @@ that the Kotlin code has been improved, but the underlying Java library remained
     <td>cloudflare</td>
     <td>4.16.0.0</td>
     <td> 
-
+ 
```

Comment:
When we dump the HTML into the README from the code instead of copy-pasting, it adds these white characters